### DISCUSSION TITLE
How to disable the automatic update on self-hosted runners?

### DISCUSSION THREAD

**Discussion by kkscilife** [community/community#50112] [Actions] [answered]
### Select Topic Area

Question

### Body

I want to disble the automatic updation on self-hosted runners because it blocks the workflow running when updating
The log sample:
Runner update in progress, do not shutdown runner.
Downloading 2.xxx runner

**Comment by vitalykarasik** [comment, marked as answer]
Didn't try yet, but check https://github.blog/changelog/2022-02-01-github-actions-self-hosted-runners-can-now-disable-automatic-updates

"If you specify the --disableupdate flag to the runner then it will not try to perform an automatic software update if a newer version of the runner is available."

**Reply by kkscilife** [reply]
Thanks! I have tryed like this: ./config.sh --url https://xxx --token xx --disableupdate. The parameter 'disableupdate' will be shown in .runner file like '"disableUpdate": true' . It's a little weird that it not works if I add the parameter directly in .runner file then restart runner.
